GeosCore/aerosol_mod.F90

```diff
@@ -808,71 +804,94 @@ SUBROUTINE AEROSOL_CONC( Input_Opt,  State_Chm, State_Diag, &
        ! Include either simple SOA (default) or Complex SOA in
        ! PM2.5 calculation.  In simulations where both Simple SOA and
        ! Complex SOA species are carried (i.e. "benchmark"), then
        ! only the Simple SOA will be added to PM2.5 and PM10, in order
        ! to avoid double-counting. (bmy, 03 Nov 2021)
+       !---------------------------------------------------------------------
        IF ( Is_SimpleSOA ) THEN
-          State_Chm%AerMass%PM25(I,J,L) = State_Chm%AerMass%PM25(I,J,L) + ( State_Chm%AerMass%SOAS(I,J,L) * ORG_GROWTH )
+          State_Chm%AerMass%PM25(I,J,L)                       =              &
+             State_Chm%AerMass%PM25(I,J,L)                    +              &
+             ( State_Chm%AerMass%SOAS(I,J,L) * ORG_GROWTH )
 
        ELSE IF ( Is_ComplexSOA ) THEN
-          State_Chm%AerMass%PM25(I,J,L) = State_Chm%AerMass%PM25(I,J,L)                 + &
-                        State_Chm%AerMass%TSOA(I,J,L)   * ORG_GROWTH  + &
-                        State_Chm%AerMass%ASOA(I,J,L)   * ORG_GROWTH  + &
-                        State_Chm%AerMass%ISOAAQ(I,J,L) * ORG_GROWTH        ! Includes SOAGX
+          State_Chm%AerMass%PM25(I,J,L)                       =              &
+             State_Chm%AerMass%PM25(I,J,L)                    +              &
+             State_Chm%AerMass%TSOA(I,J,L)   * ORG_GROWTH     +              &
+             State_Chm%AerMass%ASOA(I,J,L)   * ORG_GROWTH     +              &
+             State_Chm%AerMass%ISOAAQ(I,J,L) * ORG_GROWTH     ! Includes SOAGX
 
           ! Need to add OPOA to PM2.5 for complexSOA_SVPOA simulations
           ! -- Maggie Marvin (15 Jul 2020)
           IF ( Is_OPOA ) THEN
-             State_Chm%AerMass%PM25(I,J,L) = State_Chm%AerMass%PM25(I,J,L) + ( State_Chm%AerMass%OPOA(I,J,L) * ORG_GROWTH )
+             State_Chm%AerMass%PM25(I,J,L)                    =              &
+                State_Chm%AerMass%PM25(I,J,L)                 +              &
+                ( State_Chm%AerMass%OPOA(I,J,L) * ORG_GROWTH )
           ENDIF
        ENDIF
 
        ! Particulate matter < 10um [kg/m3]
-       State_Chm%AerMass%PM10(I,J,L) = State_Chm%AerMass%PM25(I,J,L) +                    &   ! PM2.5
-                     SOILDUST(I,J,L,5) * 0.7_fp     + &   ! + 70%  of DST2
-                     SOILDUST(I,J,L,6)              + &   ! + 100% of DST3
-                     SOILDUST(I,J,L,7) * 0.9_fp     + &   ! + 90%  of DST4
-                     State_Chm%AerMass%SALC(I,J,L)       * SSA_GROWTH
+       State_Chm%AerMass%PM10(I,J,L)                          =              &
+          State_Chm%AerMass%PM25(I,J,L)                       +              &
+          SOILDUST(I,J,L,5)                   * 0.7_fp        +              &
+          SOILDUST(I,J,L,6)                                   +              &
+          SOILDUST(I,J,L,7)                   * 0.9_fp        +              &
```

Comment:
Same comment as above regarding documentation of scale factors

Reply:
@lizziel, @1Dandan: see my comment & code block above.